pkg/redis/volumes.go

```diff
@@ -102,10 +121,45 @@ func getVolumes(r *redisv1beta1.Redis) []corev1.Volume {
 			},
 		},
 	}
+
+	if r.Spec.TLS.Enabled() {
+		svc := tls.Service{
+			SecretName: *r.Spec.TLS.GenericService.SecretName,
+			CertMount:  nil,
+			KeyMount:   nil,
+			CaMount:    nil,
+		}
+		serviceVolume := svc.CreateVolume(RedisCertPrefix)
+		vols = append(vols, serviceVolume)
+		if r.Spec.TLS.Ca.CaBundleSecretName != "" {
+			caVolume := r.Spec.TLS.Ca.CreateVolume()
+			vols = append(vols, caVolume)
+		}
+	}
+
+	return vols
+}
+
+func getTLSVolumeMounts(r *redisv1beta1.Redis) []corev1.VolumeMount {
```

Comment:
in the controller you changed it to `redisv1`, should we do this everywhere?

Reply:
Right, I'll replicate the change everywhere, I think this is the pattern that other operator are using.

Reply:
yes, then you do not have to update it everywhere when there is an api version bump